tests/test_wimprates.py

```diff
@@ -23,7 +23,7 @@ def test_elastic():
     isclose(wr.rate_wimp_std(1, **opts), ref)
 
     # Test numericalunits.reset_units() does not affect results
-    nu.reset_units()
+    nu.reset_units("SI")
```

Comment:
Did you add this on purpose, e.g. to make the tests deterministic? I think the point of the test is that the unit system should not affect rate_wimp_std.

Reply:
indeed-- this was when I was trying to realise what numericalunits does to default arguments, 
it is a grace changing it back does not alter the tests.